cmake/FindLibBcc.cmake

```diff
@@ -60,8 +60,15 @@ if(STATIC_LINKING)
   # If we do a static bpftrace build, we must link them in.
   find_package(LibBpf)
   find_package(LibElf)
-  find_package(LibZ)
-  SET(CMAKE_REQUIRED_LIBRARIES ${LIBBCC_BPF_LIBRARIES} ${LIBBPF_LIBRARIES} ${LIBELF_LIBRARIES} ${LIBZ_LIBRARIES})
+
+  if(${CMAKE_SYSTEM_NAME} STREQUAL "Android")
```

Comment:
I don't think you have to interpolate this.
`if (CMAKE_SYSTEM_NAME STREQUAL "Android")` should work?
Even better, the ndk toolchain defines `ANDROID`, so `if (ANDROID)` should be enough.